frontend/src/components/c-ramp.vue

```diff
@@ -2,7 +2,7 @@
 .ramp
   template(v-if="tframe === 'commit'")
     template(v-for="(slice, j) in user.commits")
-      template(v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)")
+      template(v-for="(commit, k) in slice.commitResults")
```

Comment:
Can I ask whether it is safe to remove this check and include empty commits? Another question here is whether merge commits can be designed to be empty commits. The reason for this question is, certain merge commits may introduce additional changes when resolving conflicts, which I think is reasonable to be reflected on the dashboard. In this case, merge commits should still have the actual number of additions and deletions and these should get reflected in the tooltip and zoom panel.